tests/src/ObjectMapping.cc

```diff
@@ -0,0 +1,89 @@
+#include "ObjectMapping.h"
+
+#include "EVENT/LCEvent.h"
+#include "EVENT/Track.h"
+#include "EVENT/TrackerHit.h"
+#include "EVENT/TrackerHitPlane.h"
+#include "EVENT/SimTrackerHit.h"
+#include "EVENT/CalorimeterHit.h"
+#include "EVENT/RawCalorimeterHit.h"
+#include "EVENT/SimCalorimeterHit.h"
+#include "EVENT/TPCHit.h"
+#include "EVENT/Cluster.h"
+#include "EVENT/Vertex.h"
+#include "EVENT/ReconstructedParticle.h"
+#include "EVENT/MCParticle.h"
+#include "EVENT/LCEvent.h"
+#include "EVENT/LCCollection.h"
+#include "UTIL/LCIterator.h"
+
+#include "edm4hep/TrackCollection.h"
+#include "edm4hep/TrackerHitCollection.h"
+#include "edm4hep/TrackerHitPlaneCollection.h"
+#include "edm4hep/SimTrackerHitCollection.h"
+#include "edm4hep/ClusterCollection.h"
+#include "edm4hep/CalorimeterHitCollection.h"
+#include "edm4hep/RawCalorimeterHitCollection.h"
+#include "edm4hep/SimCalorimeterHitCollection.h"
+#include "edm4hep/MCParticleCollection.h"
+#include "edm4hep/ReconstructedParticleCollection.h"
+#include "edm4hep/RawTimeSeriesCollection.h"
```

Comment:
No problem for me simply removing all the compatibility things here (maybe in a follow up PR) and explicitly depending on a recent enough version of EDM4hep.

Reply:
Ah, I now understand this comment fully. This commit would break compatibility, while the rest has it. I have no strong opinion on making this compatible and explicitly breaking it later, or opening another PR that does the breaking so that this doesn't have to any longer.

Reply:
This is already broken now in the main branch, so I don't see a reason to make this PR compatible. I have opened https://github.com/key4hep/k4EDM4hep2LcioConv/issues/30 to keep track of this.